src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  box-sizing: border-box;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. If the user's system doesn't have the 'Roboto' font, the browser will use the next available font in the list. If none of the specified fonts are available, then it will use the default system font. This can cause your website to look different than you intended. You have provided 'Arial', 'Helvetica', and 'sans-serif' as fallbacks, which is good, but you can improve this by including a generic family name at the end of the list. For example: `font-family: Roboto, Arial, Helvetica, sans-serif, sans-serif;` The generic family names are 'serif', 'sans-serif', 'cursive', 'fantasy', and 'monospace'.